Integrations/integration-Cylance_Protect_v2.yml

```diff
@@ -173,6 +179,15 @@ script:
         ''' Translates API raw float (-1 to 1) score to UI score (-100 to 100) '''
         return score*100
 
+    def translate_score(score, threshold):
+        if score > 0:
```

Comment:
Note that the threshold should determine if the indicator is malicious. 
If the score is higher or equal to the threshold then the DBot score should be 3 (malicious). 
Usually, if the score is in the range of threshold/2-threshold, the DBot score given is 2 (suspicious).
I don't know if this matches your case, but please consider this and match the logic.

Reply:
It is not the case here, as for malicious/suspicious threats - the threat score will be negative.